mitodl/mitxpro
# Add readable_id to ProductVersion

#### Pre-Flight checklist

#### What are the relevant tickets?
 Closes #446

#### What's this PR do?
Adds a `readable_id` to `ProductVersion` and overrides its `save` method to populate it based on the underlying `CourseRun.courseware_id` or `Program.readable_id`.

#### How should this be manually tested?
- After the migration script is run, most or all `ProductVersion` objects should have a populated `readable_id` field.
- if you create a new `ProductVersion`, the `readable_id` field should be correctly populated.
- if you change the associated `product` for a `ProductVersion` and save, the `readable_id` field should be changed to reflect that.
